Makefile

```diff
@@ -188,5 +192,24 @@ clean:
 deploy_kind: container
 	kind load docker-image --name $(KIND_CLUSTER) $(REGISTRY)/$(TARGET):$(IMAGE_VERSION) || true
 
+# kind_images will build the kind-node image. Generally building the base image is not necessary
+# and we can use the upstream kindest/base image.
+kind_images: check-kind-env
+	kind build node-image --kube-root=$(K8S_PATH) --image $(REGISTRY)/kind-node:$(KIND_K8S_TAG)
+
+# push_kind_images will push the same image kind_images just built our registry.
+push_kind_images:
+	docker push $(REGISTRY)/kind-node:$(KIND_K8S_TAG)
+
+# check-kind-env will show you what will be built/tagged before doing so with kind_images
+check-kind-env:
+ifndef K8S_PATH
+	$(error K8S_PATH is undefined)
+endif
+ifndef KIND_K8S_TAG
+	$(error KIND_K8S_TAG is undefined)
+endif
+	echo --kube-root=$(K8S_PATH) tagging as --image $(REGISTRY)/kind-node:$(KIND_K8S_TAG)
```

Comment:
Was this a debugging line? I think the output from make is verbose and will show the `kind build node-image...` command details. If you wanted to include it, it might be better as part of the `kind_images` target as that is where the building/tagging takes place.

Reply:
Well it was partially a debug line and partially a helper since I thought it might make sense in our steps to do something like `make check-kind-env` and verify the path and versions are what we want. The output from the build starting isn't actually that great at the start; once things are tagged all is clear, but its not that optimal to either waste that time or even start/stop that mid build IMO.